.travis.yml

```diff
@@ -11,7 +11,6 @@ script:
 - bundle exec rake arm:build
 - unset BUNDLE_GEMFILE
 - cd $TRAVIS_BUILD_DIR/runtime/ms_rest && bundle install --gemfile=Gemfile && bundle exec rspec
-- unset BUNDLE_GEMFILE
```

Comment:
@veronicagg I verified that there is not need to keep two unset as running bundle install from script section of the travis doesn't set or affect that environment variable.